README.md

```diff
@@ -26,8 +26,9 @@ When building a containerized FIPS-compliant app, in general we recommend using
 
 To view the full list of available Go tags in MAR:
 
-* Visit the [AZCU Indexer](https://azcuindexer.azurewebsites.net/repositories/oss/go/microsoft/golang) (*Microsoft internal auth required*)
-  * Click on a tag name to copy the full tag URL to your clipboard
+* Visit [`golang` on the MAR Discovery Portal][MAR] (*Microsoft internal auth required*)
+  * You must be currently signed into the MAR discovery portal with Microsoft internal auth to see the golang image. If you see a "not found" page, make sure you are signed in at the top-right corner of the MAR page, **then close the tab and click the [`golang`][MAR] link again**.
```

Comment:
Filed https://github.com/microsoft/mcr/issues/2514 (private repo) to try to get this flow improved.